tests/Persistence/Mapping/ColocatedMappingDriverTest.php

```diff
@@ -61,7 +76,18 @@ public function testGetAllClassNames(string $path): void
 
         $classes = $driver->getAllClassNames();
 
-        self::assertSame([TestClass::class], $classes);
+        sort($classes);
+        self::assertSame([Entity::class, EntityFixture::class], $classes);
+    }
+
+    public function testGetAllClassNamesWithRegex(): void
+    {
+        $noFixturesRegex = '/^(?!.*Fixture\.php$).*\.php$/';
+        $driver          = $this->createDriver(__DIR__ . '/_files/colocated', $noFixturesRegex);
+
+        $classes = $driver->getAllClassNames();
+
+        self::assertSame([Entity::class], $classes);
```

Comment:
> I think it should be quite obvious that if testGetAllClassNames asserts about [Entity::class, EntityFixture::class], while new test asserts only about [Entity::class], this means that fixture has been excluded by regex.

You still feel the need to mention it though. Let's make it even more obvious then.

```suggestion
        self::assertSame(
                [Entity::class],
                $classes,
                'EntityFixture.php should be excluded by the regular expression'
        );
```